### PR TITLE
SWC-6843

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java [hunk 1]
```diff
@@ -1551,8 +1551,9 @@ private void configureEditProjectMetadataAction() {
   }
 
   private void configureProjectHelpAction() {
-    boolean isProject = entityBundle.getEntity() instanceof Project;
-    actionMenu.setActionVisible(Action.PROJECT_HELP, isProject);
+    boolean isProjectMenu =
+      entityBundle.getEntity() instanceof Project && currentArea == null;
+    actionMenu.setActionVisible(Action.PROJECT_HELP, isProjectMenu);
```
Comment (xschildw) [inline src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java:1556]: Do we need to add to the menu? There's a info/help icon next to Project that's always there.

Comment (jay-hodgson) [reply src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java:1556]: The "menu" includes this info/help icon.  It should probably be called an ActionBar or something at this point.  It started out as a simple menu

Comment (jay-hodgson) [reply src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java:1556]: This visibility controls that info/help icon to show up next to the dropdown menu!

Comment (jay-hodgson) [reply src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java:1556]: See this PR for more information: https://github.com/Sage-Bionetworks/SynapseWebClient/pull/5379

**Review by xschildw — APPROVED** [review]

**Pull request merged**